src/Backend/Modules/ContentBlocks/Layout/Templates/ContentBlockEdit.html.twig

```diff
@@ -51,7 +51,7 @@
     <div class="col-md-12">
       <div class="btn-toolbar">
         <div class="btn-group pull-left" role="group">
-          {% if showContentBlocksDelete %}
+          {% if showContentBlocksContentBlockDelete %}
```

Comment:
idem, this actually points out nicely why in this case it is a bad idea, this is almost unreadable